analyzer/src/funTest/assets/projects/external/example-python-flask-expected-output.yml

```diff
@@ -33,8 +33,9 @@ packages:
     purl: "pkg:pypi/Flask@0.12"
     declared_licenses:
     - "BSD"
+    - "BSD License"
     declared_licenses_processed:
-      spdx_expression: "BSD-3-Clause"
+      spdx_expression: "BSD-3-Clause AND BSD-2-Clause"
```

Comment:
Wow, does that mean we map `BSD` to `BSD-3-Clause` but `BSD License` to `BSD-2-Clause`? If so, that sounds like a bug to me, even if unrelated to this PR.

Reply:
I'm going to fix that.

Reply:
Mind rebasing your PR on https://github.com/oss-review-toolkit/ort/pull/2408 once it's merged to see the changes in `spdx_expression` be gone?

Reply:
Done.

Reply:
@sschuberth  

> Wow, does that mean we map `BSD` to `BSD-3-Clause` but `BSD License` to `BSD-2-Clause`? If so, that sounds like a bug to me, even if unrelated to this PR.

@sschuberth  No it actually correct - based on our experience most of the time when people state "BSD" they mean BSD-3-Clause and BSD License is commonly older OSS where BSD-2-Clause was more common,
We should make a decision for these kind of cases - do we always pick the stricter (BSD-3-Clause) or the weaker (BSD-2-Clause), or alternatively do we go what is the most popular (BSD-3-Clause)

Reply:
In PR https://github.com/oss-review-toolkit/ort/pull/2408 I already went for the "go what is the most popular"-approach, because as stated there, for all of our test project that use "BSD License" they were actually meaning "BSD-3-Clause".

In any case, was quite obscure to map "BSD" to one thing but "BSD License" to something else. It's good to have that aligned now.